src/QuadForm/ClosestVector.jl

```diff
@@ -69,20 +59,26 @@ function pojective_quadratic_triple(Q::MatrixElem, L::MatrixElem, c::RingElement
         push!(pp1, p1)
         push!(LL1, L1)
         push!(r1, Q[n+1-i, n+1-i])
-        push!(m1, L[n+1-i,1:1])
+        push!(m1, L[n+1-i,1:1][1])
         pL =[Lqqt, pp1, LL1, r1, m1]
     end
     #------------------------------------------------
+    c1 = c; pQT = []; ppQT = []; pLL = []; 
+    ppLL = []; cc1 = []; Lqt = [];
     for j in 1:size(pL[1],1)
```

Comment:
So `pL[1] === Lqqt` by definition, so why not use this? What is the point of `pL`

Reply:
That is just me being lazy and I didn't want to remember the names of different lists in the function. But I can change it.